index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./src/style/style.css">
+    <script src="https://kit.fontawesome.com/89d87f2112.js" crossorigin="anonymous"></script>
+    <title>Programming courses</title>
+</head>
+
+<body>
+    <nav>
+        <div class="navbarContainer">
+            <div class="hamburgerButton">
+                <img class="hamburgerButtonImg" src="./src/icon/hamburgerButton.png" alt="">
+            </div>
+            <div class="logo">
+                <a href="index.html"><img src="./src/icon/Logo.png" alt=""></a>
+            </div>
+            <ul class="links">
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#courses">courses</a></li>
+                <li><a href="./index.html#Sponsor">Sponsor</a></li>
+                <li class="button"><a href="">Campaing</a></li>
+            </ul>
+        </div>
+        <div class="menu menuTwo">
+            <div class="menuBox">
+                <div class="menuLogo">
+                    <img src="./src/icon/Logo.png" alt="">
+                    <i class="fa-solid fa-xmark"></i>
+                </div>
+                <ul class="menuLinks">
+                    <li><a  href="./about.html">About</a></li>
+                    <li><a  href="#courses">Courses</a></li>
+                    <li><a href="#Sponsor">Sponsor</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
```

Comment:
- [ ] You did a fantastic job on the navigation 👏. To ensure that you stay true to the original design, I recommend adding the top navigation bar with the black background color. Please also include the orange borders around the **Campaing*** link. :ok_hand:  🤓 
### Your design  :point_down: 
![Screenshot from 2023-08-03 00-08-52](https://github.com/ammarAlaliS/Capstone-project_CohetePage.github.oi/assets/89228743/775ba370-05f4-457b-86dd-fc7698489313)



### The original design :point_down: 
![Screenshot from 2023-08-02 23-44-30](https://github.com/ammarAlaliS/Capstone-project_CohetePage.github.oi/assets/89228743/0d5e2a10-e394-4e26-a5a0-e899e03d9c7b)